target/linux/ramips/dts/mt7621_ruijie_rg-ew1200g-pro-v1.1.dts

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+    compatible = "mediatek,mt7621-soc", "ruijie,rg-ew1200g-pro-v1.1";
```

Comment:
Please use tabs instead of spaces for indentation.

Reply:
Done